hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/states/TestNodeStateMap.java

```diff
@@ -127,26 +129,26 @@ public void testGetNodeMethodsReturnCorrectCountsAndStates()
    */
   @Test
   public void testConcurrency() throws Exception {
-    NodeStateMap nodeStateMap = new NodeStateMap();
+    //NodeStateMap nodeStateMap = new NodeStateMap();
```

Comment:
This commented line should probably be removed, however I am not sure if any of these NodeStateMap changes are needed, as I am not sure if we need to add the node to the topology map in NodeStateMap, when it already happens in the register method.

Reply:
Yes, this line should be removed.